schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,159 @@
+{
+    "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
```

Comment:
a general comment is that all properties in the schema type are defined explicitly, this will cause any template expression be denied by VS, for example, "[parameters("foo")]" will be marked as error for a property value for a trafficmanager resource in template.
I suggest for some properties we expect user will use parameters or other expressions, use "oneOf" and "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression", you can take schemas/2015-08-01/Microsoft.Cache.json as example.